src/styles/scss/components.scss

```diff
@@ -1,22 +1,33 @@
 @import url("https://fonts.googleapis.com/css2?family=Questrial&display=swap");
 
+:root {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```

Comment:
probably not necessary if the main App.css funnels these defaults through the whole app